webapp/pages/my-groups.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <div>my groups</div>
+    <group-teaser />
+    <br />
+    <br />
+    <group-card :items="responseGroupListQuery" />
+  </div>
+</template>
+<script>
+import GroupTeaser from '~/components/Group/GroupTeaser.vue'
+import GroupCard from '~/components/Group/GroupCard.vue'
```

Comment:
`GroupTeaser` and `GroupCard` is not named correctly here, but we can care for that later on …